languages/ruby/reference/exercise-concepts/nucleotide-count.md

```diff
@@ -0,0 +1,53 @@
+# Concepts of nucleotide-count
+
+## General
+
+- class level methods: used as the starting point of the exercise. They may be named with `self.[method_name]` notation or with a `self` block. Another way of creating class methods is extending a module.
```

Comment:
Since `method_name` is used as a method name in some popular gems, should we change how this is presented as a place holder?

`self.some_name` I think is clear... and to a Rubyists it is clear that it must be a method, as opposed to a keyword or a variable, as neither will use an explicit receiver. (I could be incorrect or outright wrong about the keywords, but variables aren't invoked in this way).